dashboard/src/actions/repos.test.tsx

```diff
@@ -248,6 +252,27 @@ describe("installRepo", () => {
       const res = await store.dispatch(installRepoCMDAuth);
       expect(res).toBe(true);
     });
+
+    context("when a pod template is provided", () => {
+      const installRepoCMDPodTemplate = repoActions.installRepo(
+        "my-repo",
+        "http://foo.bar",
+        "",
+        "",
+        "foo: bar",
```

Comment:
This is not a real example, right? I see we've some examples in our docs so maybe we can reuse them as the content for this argument:

```
apiVersion: kubeapps.com/v1alpha1
kind: AppRepository
metadata:
  name: my-repo
  namespace: kubeapps
spec:
  syncJobPodTemplate:
    metadata:
      labels:
        my-repo: "isPrivate"
    spec:
      containers:
        - env:
            - name: FOO
              value: BAR
  url: https://my.charts.com/```